src/consts/wallet.ts

```diff
@@ -216,6 +216,7 @@ export const SANCTIONED_WALLETS = Object.freeze(
     'TTgcTTNbNuFdbrhvbjMZVrdU5KALyzDaPw',
     'TTKnV2S1295UrPr7N67Tp9ykNL7xX2Z4Uj',
     'TTUDyVhhpCC1xJoPmWzdjLAzeoPwbSABdr',
+    'TU4tDFRvcKhAZ1jdihojmBWZqvJhQCnJ4F',
```

Comment:
This is also getting added in the other PR https://github.com/wormhole-foundation/wormhole-connect/pull/3625/files. Maybe adding of the sanctioned wallet should be its own PR?